From d81e10b63f5a38a5004ff2486e679608cb7dbab9 Mon Sep 17 00:00:00 2001
From: Pascal de Bruijn <pmjdebruijn@pcode.nl>
Date: Sat, 29 May 2010 05:54:58 +0000
Subject: profiles: Use lamemp3enc and xingmux instead of the deprecated lame element

The lame element has a broken interface and results in worse
quality encodings. The xingmux element is added to allow correct
duration reporting and seeking for VBR files.

Fixes bug #619642.
---
diff --git a/profiles/gnome-audio-profiles.schemas.in.in b/profiles/gnome-audio-profiles.schemas.in.in
index bd5ab06..d29ac57 100644
--- a/profiles/gnome-audio-profiles.schemas.in.in
+++ b/profiles/gnome-audio-profiles.schemas.in.in
@@ -496,7 +496,7 @@
       <applyto>/system/gstreamer/@GST_MAJORMINOR@/audio/profiles/mp3/pipeline</applyto>
       <owner>gnome-media</owner>
       <type>string</type>
-      <default>audio/x-raw-int,rate=44100,channels=2 ! lame name=enc mode=0 vbr-quality=6 ! id3v2mux</default>
+      <default>audio/x-raw-int,rate=44100,channels=2 ! lamemp3enc name=enc target=0 quality=6 ! xingmux ! id3v2mux</default>
       <locale name="C">
         <short>The partial GStreamer pipeline used</short>
         <long>
--
cgit v0.8.3.1
